### PR TITLE
docs/variants/protectli_vp46xx: document ath10k firmware issue

### PR DESCRIPTION
**Pull request by pkubaj**
Signed-off-by: Piotr Kubaj <piotr.kubaj@3mdeb.com>

### CODE REVIEW

File: docs/variants/protectli_vp46xx/overview.md [hunk 1]
```diff
@@ -37,3 +37,4 @@ For more information please refer to the references below.
     validation procedure.
 * [Test matrix](test-matrix.md) - describes validation scope used during
     Dasharo firmware validation procedure.
+* [Post-installation setup](post-install.md) -
```
Comment (macpijan) [inline docs/variants/protectli_vp46xx/overview.md:40]: @pkubaj Either the `-` at the end is not required, or some description after should be added?

**Review by macpijan — CHANGES_REQUESTED** [review]
I would fix this myself, it it would be a branch in dasharo repo, not a fork

File: docs/variants/protectli_vp46xx/overview.md (hunk 1 above)
Comment (miczyg1) [reply docs/variants/protectli_vp46xx/overview.md:40]:
```suggestion
* [Post-installation setup](post-install.md)
```

**Comment by miczyg1** [comment]
> I would fix this myself, it it would be a branch in dasharo repo, not a fork

@macpijan there is a w/a to that. Simply use suggestion via GH UI and it lets you apply the suggestion on a fork :)

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**